ui/src/app/pages/admin/admin-notebook-view.tsx

```diff
@@ -0,0 +1,109 @@
+import {SpinnerOverlay} from 'app/components/spinners';
+import {workspaceAdminApi} from 'app/services/swagger-fetch-clients';
+import colors from 'app/styles/colors';
+import {reactStyles} from 'app/utils';
+import {reactRouterUrlSearchParams} from 'app/utils/navigation';
+import * as React from 'react';
+import {useEffect, useState} from 'react';
+import {useParams} from 'react-router';
+
+const styles = reactStyles({
```

Comment:
these are copied from a variety of sources.  The goal here is "looks OK" rather than polished